src/com/projectkorra/projectkorra/firebending/lightning/Lightning.java

```diff
@@ -218,7 +466,25 @@ public void progress() {
 					if (target != null) {
 						this.destination = target.getLocation();
 					} else {
-						this.destination = this.player.getEyeLocation().add(this.player.getEyeLocation().getDirection().normalize().multiply(this.range));
+						Block targetBlock = GeneralMethods.getTargetedLocation(this.player, this.range, false).getBlock();
+						boolean foundCopper = isCopper(targetBlock), foundRod = isLightningRod(targetBlock);
+						
+						if (!foundCopper) {
+							if (!foundRod) {
+								for (Block block : GeneralMethods.getBlocksAroundPoint(targetBlock.getLocation(), 1.25)) {
+									if (isLightningRod(block)) {
+										foundRod = true;
+										targetBlock = block;
+										break;

```

Comment:
Why do you only search for lightning rods in a radius and not copper?

Reply:
didn't seem necessary to search for copper in a radius. lightning rods have a specific bounding box which makes it pretty annoying to get its location, might honestly resort to using raytracing. the only reason i included copper here actually was because of how inaccurate lightning is to reaching its destination lol, so i made it easier for players to be able to shoot at copper to conduct

Reply:
It just seems odd to me that they can only hit and chain in copper if they aim at it. Same with lightning rods

Reply:
copper and lightning rods dont attract lightning, and it doesnt really make sense for lightning to automatically be sent to a location you dont want it to go just because theres a copper block or lightning rod nearby

Reply:
I guess that's a valid point